snapshotter_opts_unix.go

```diff
@@ -20,17 +20,93 @@
 package containerd
 
 import (
+	"context"
 	"fmt"
 
 	"github.com/containerd/containerd/snapshots"
 )
 
+const (
+	labelSnapshotUIDMapping = "containerd.io/snapshot/uidmapping"
+	labelSnapshotGIDMapping = "containerd.io/snapshot/gidmapping"
```

Comment:
These constants can be public.

Probably should be in the `containerd/snapshots` pkg.

Reply:
@dgl Wanna fix this? :)

Reply:
Done.